### PR TITLE
Change separator for client properties and change client name in scripts

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

**Comment by BrennanConroy** [comment]
🆙 📅

File: src/BenchmarksDriver/README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Options:
   --header               Header added to request.
   --headers              Default set of HTTP headers added to request (None, Plaintext, Json, Html). Default is Html.
   --method               HTTP method of the request. Default is GET.
-  --properties           Key value pairs of properties specific to the client running. e.g., ScriptName=pipeline,PipelineDepth=16"
+  -p|--properties        Key value pairs of properties specific to the client running. e.g., -p ScriptName=pipeline -p PipelineDepth=16"
```
Comment (sebastienros) [inline src/BenchmarksDriver/README.md]: Missing or extraneous quotes

File: src/BenchmarksDriver/Program.cs [hunk 2]
```diff
@@ -521,10 +521,9 @@ public static int Main(string[] args)
                 }
                 if (clientProperties.HasValue())
                 {
-                    var properties = clientProperties.Value().Split(',');
-                    foreach (var kvp in properties)
+                    foreach (var property in clientProperties.Values)
                     {
-                        var values = kvp.Split('=');
+                        var values = property.Split('=');
```
Comment (sebastienros) [inline src/BenchmarksDriver/Program.cs]: Prefer `IndexOf` as in environment variables as we could have values with `=`.

File: src/Benchmarks/benchmarks.html.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "Default": {
 
-    "ClientName": "wrk",
+    "Client": "wrk",
```
Comment (sebastienros) [inline src/Benchmarks/benchmarks.html.json]: Pretty please, fix the `wrk` case just in case ❤️

**Comment by BrennanConroy** [comment]
Done

**Pull request merged**